crates/node/src/service.rs

```diff
@@ -244,10 +271,24 @@ where
 }
 
 /// Builds a new service for a full client.
+///
+/// Constructs a full client service based on the provided configuration. It sets up essential
+/// components such as the backend, transaction pool, import queue, and more.
+///
+/// # Arguments
+///
+/// * `config` - A `Configuration` object that contains the service configuration.
+/// * `sealing` - An optional `Sealing` value that specifies the sealing mode to use.
+///
+/// # Returns:
+///
+/// * A `Result` containing a `TaskManager` object if the service was successfully created, or a `ServiceError` if an error occurred.
 pub fn new_full(config: Configuration, sealing: Option<Sealing>) -> Result<TaskManager, ServiceError> {
     let build_import_queue =
         if sealing.is_some() { build_manual_seal_import_queue } else { build_aura_grandpa_import_queue };
 
+    // Create a partial service by calling `new_partial` function with the chosen import queue function.
+    // This step sets up essential components for the full client, including the backend, transaction pool, import queue, and more.
```

Comment:
`including the backend, transaction pool, import queue, and more.`
This is already expressed by the code